scripts/data/generate_words2.py

```
@@ -0,0 +1,329 @@
+    ("specific","具体的な","spɪˈsɪfɪk","具体的な"),
+    ("successful","成功した","səkˈsesfəl","成功した"),
+    ("sufficient","十分な","səˈfɪʃənt","十分な"),
+    ("temporary","一時的な","ˈtempəreri","一時的な"),
+    ("various","様々な","ˈveriəs","様々な"),
+]
+
+# Additional nouns
+extra_nouns = [
+    ("approach","アプローチ","əˈproʊtʃ","アプローチ"),
+    ("benefit","利益","ˈbenɪfɪt","利益"),
+    ("challenge","挑戦","ˈtʃælɪndʒ","挑戦"),
```

Ruff (0.15.9)

[warning] String contains ambiguous `ˈ` (MODIFIER LETTER VERTICAL LINE). Did you mean ``` (GRAVE ACCENT)?

(RUF001)